react/Checkbox/Checkbox.less

```diff
@@ -40,16 +40,16 @@
     .focus();
   }
 
-  .input:checked + .label & {
-    background-color: @sk-mid-gray-light;
+  .label & {
+    background-color: @sk-white;
   }
 }
 
 .checkMark {
   display: block;
   width: 100%;
   height: 100%;
-  fill: @sk-black;
+  fill: @sk-focus;
```

Comment:
That's fine if they are deliberately in sync at the moment. But going with `@sk-form-accent` future proofs the api. You can set `@sk-form-accent: @sk-focus` to allow the flexibility to change this independently later.